addons/package_hierarchy/models/stock_quant_package.py

```diff
@@ -61,9 +61,12 @@ def _compute_children_quant_ids(self):
     @api.depends('quant_ids.package_id', 'quant_ids.location_id', 'quant_ids.company_id', 'quant_ids.owner_id')
     def _compute_package_info(self):
         for package in self:
-            values = {'location_id': False, 'company_id': self.env.user.company_id.id, 'owner_id': False}
-            package._check_not_multi_location()
-            values['location_id'] = package.children_quant_ids.mapped('location_id')
+            values = {
+                'location_id': False, 'company_id': self.env.user.company_id.id, 'owner_id': False
+            }
+            locations = package.children_quant_ids.mapped('location_id')
```

Comment:
Can we disable prefetching here? Just to be safe.

`locations = package.with_context(prefetch_fields=False).children_quant_ids.mapped('location_id')`

Reply:
I think should be fine